apps/admin/frontend/src/screens/unconfigured_screen.tsx

```diff
@@ -71,64 +54,56 @@ export function UnconfiguredScreen(): JSX.Element {
     []
   );
   const [isLoading, setIsLoading] = useState(false);
-  const [vxElectionFileIsInvalid, setVxElectionFileIsInvalid] = useState(false);
-  const [systemSettingsFileIsInvalid, setSystemSettingsFileIsInvalid] =
-    useState(false);
+  const [configureError, setConfigureError] = useState<ConfigureError>();
   const [isUsingConverter, setIsUsingConverter] = useState(false);
 
   const client = useMemo(
     () => getElectionDefinitionConverterClient(converter),
     [converter]
   );
 
-  const saveSystemSettingsToBackend = useCallback(
-    (fileContent: string) => {
-      setSystemSettingsMutation.mutate(
-        { systemSettings: fileContent },
-        {
-          onSuccess: (result) => {
-            if (result.isErr()) {
-              setSystemSettingsFileIsInvalid(true);
-            }
-          },
+  const configureMutateAsync = configureMutation.mutateAsync;
+  const configureFromElectionPackage = useCallback(
+    async (
+      electionData: string,
+      systemSettingsData: string = JSON.stringify(DEFAULT_SYSTEM_SETTINGS)
+    ) => {
+      setConfigureError(undefined);
+      try {
+        const result = await configureMutateAsync({
+          electionData,
+          systemSettingsData,
+        });
+        if (result?.isErr()) {
+          setConfigureError(result.err());
+          // eslint-disable-next-line no-console
+          console.error(
+            'configure failed in saveElectionToBackend',
+            result.err().message
+          );
```

Comment:
Curious what your thinking is around `console.error` logs on the frontend instead of, say, using a frontend `Logger` or backend logging

Reply:
I just maintained what was already there 🤷

Reply:
Ah missed that!